arm/ripper/logger.py

```diff
@@ -23,9 +25,22 @@ def setuplogging(job):
         logfile = job.label + ".log"
 
     if cfg['LOGPATH'][-1:] == "/":
-        logfull = cfg['LOGPATH'] + logfile
+        #Check to see if file already exists, if so, create a new file
+        TmpLogFull = cfg['LOGPATH'] + logfile
+        if os.path.isfile(TmpLogFull):
+            logfile = str(job.label) + "_" + str(round(time.time() * 100)) + ".log"
+            logfull = cfg['LOGPATH'] + logfile
+        else:
+            logfull = cfg['LOGPATH'] + logfile       
     else:
-        logfull = cfg['LOGPATH'] + "/" + logfile
+        #Check to see if file already exists, if so, create a new file
+        TmpLogFull = cfg['LOGPATH'] + "/" + logfile
+        if os.path.isfile(TmpLogFull):
+            #CTime = round(time.time() * 100)
+            logfile = str(job.label) + "_" + str(round(time.time() * 100)) + ".log"
+            logfull = cfg['LOGPATH'] + "/" + logfile
+        else:
+            logfull = cfg['LOGPATH'] + "/" + logfile
```

Comment:
Not blocking, but this if else statement looks like it could be consolidated to a ternary statement for the TmpLogFull string.

Reply:
This doesnt check if it's the empty.log .This caused my some logging issues and would break the arm empty.log

I had to add a if statement to avoid having a broken log files
`    if logfile != "empty.log":`

Reply:
oh snap. I was wondering what happened to that. @133-server, do you want to include this fix, in your upcoming pull request? I have to do one for a blu-ray fix, so I am happy to copy and paste the above code into mine.

Reply:
Sorry i didnt see the notification for this.

Its up to you.  Ive been having alot of issues with the logging tbh. i have changed some of the code, I'm trying to change the logging to have a consistant logging when the script fails. In my experience; ARM if it has issues will just die with no logging. and i have to try running the script via ssh. Im trying to have it so that the NAS log becomes the main log file if there's any problems.

The apprise notifications kind of go around this issue, but im trying to find an easier solution for those who dont have apprise setup.